packages/block-editor/src/components/block-list/style.scss

```diff
@@ -104,8 +100,24 @@
 			border-color: var(--wp-admin-theme-color);
 		}
 	}
-}
 
+	// Ensure an accurate partial text selection.
+	// To do this, we disable text selection on the main container, then re-enable it only on the
+	// elements that actually get selected.
+	// To keep in mind: user-select is currently inherited to all nodes inside.
+
+	// This is a hack to only disable user-select when blocks are selected inside.
+	// @todo: We shouldn't be targetting the label, this is only a POC. Should be replaced with a class.
+	//user-select: none;
+	[aria-label="Multiple selected blocks"] & {
```

Comment:
This is obviously in need of a change.

But what I did here was to scope the disabling of `user-select: none;` to only ever apply when a block is selected. I don't know if it's needed — if it is, we should assign a classname on this element instead — but it does _seem_ safer. @ellatrix what do you think?